File: api/REC.Model/Channel.cs
using Dapper.Contrib.Extensions;
using REC.Model.Enum;

namespace REC.Model
{
    [Table("Channel")]
    public class Channel
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public string Path { get; set; }
        public string Folder { get; set; }
        public bool Active { get; set; }
        public string Image { get; set; }
        public ChannelType ChannelType { get; set; }
    }
}

File: admin/src/app/shared/models/channel.ts
export interface Channel {
    active: boolean;
    channelType: number;
    createdAt: string;
    id: any;
    name: string;
    path: string;
    image?: string;
}

export const ChannelEnum = {
    Radio: 1,
    TV: 2,
};

export interface ChannelType {
  id: number;
  label: string;
}

export interface ActiveValues {
  id: boolean;
  name: string;
}

export interface ChannelPaginate {
  result: Channel[];
  pageIndex: number;
  pageSize: number;
  total: number;
}
